src/components/views/auth/index.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import * as AuthSession from 'expo-auth-session'
+import * as WebBrowser from 'expo-web-browser'
+import jwtDecode from 'jwt-decode'
+import { Alert, Platform, Text } from 'react-native'
+import styled from 'styled-components/native'
+import { useTranslation } from 'react-i18next'
+import Container from 'components/atoms/container'
+import Button from 'components/atoms/button'
+
+WebBrowser.maybeCompleteAuthSession()
+
+const Avatar = styled.Image`
+  width: 100px;
+  height: 100px;
+  margin-bottom: 20px;
+`
+
+const useProxy = Platform.select({ web: false, default: true })
+const redirectUri = AuthSession.makeRedirectUri({ useProxy })
+
+export const Auth = () => {
+  const [data, setData] = React.useState(null)
+  const { t } = useTranslation()
+
+  const [request, result, promptAsync] = AuthSession.useAuthRequest(
+    {
+      redirectUri,
+      clientId: process.env.AUTH0_CLIENT_ID,
+      // id_token will return a JWT token
+      responseType: 'id_token',
+      // retrieve the user's profile
+      scopes: ['openid', 'profile'],
+      extraParams: {
+        // ideally, this will be a random value
+        nonce: 'nonce'
```

Comment:
Due to time constraints could we have a new issue to do this. guidelines here https://auth0.com/docs/api-auth/tutorials/nonce